src/openrct2/platform/shared.c

```diff
@@ -122,6 +127,12 @@ void platform_update_palette(const uint8* colours, sint32 start_index, sint32 nu
 
 void platform_init()
 {
+#ifdef _WIN32 
+    LARGE_INTEGER freq;
+    QueryPerformanceFrequency(&freq);
```

Comment:
`platform_init` is deleted in one of my refactor branches. So please don't put anything in it. As @janisozaur suggested, use `core_init`.